CHANGELOG.md

```diff
@@ -366,7 +435,8 @@ See the comparison for full list.
 
 Initial tagged release.
 
-[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.11.0...HEAD
+[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.12.0...HEAD
```

Comment:
Since we only deploy to `master` on a new release, then yes, comparing `master...HEAD` would work.